src/test/java/org/springframework/data/elasticsearch/client/reactive/ReactiveElasticsearchClientUnitTests.java

```diff
@@ -577,6 +582,83 @@ public void searchShouldReturnEmptyFluxIfNothingFound() {
 				.verifyComplete();
 	}
 
+	// --> AGGREGATE
```

Comment:
when we do not request the source, I think we don't need the test cases checking for the source (and the result json files)

Reply:
Committed a change for this.